app/src/docs/Rating.doc.tsx

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
 import {
-    BaseDocsBlock, DocExample, EditableDocContent, UUI3, UUI4,
+    BaseDocsBlock, DocExample, EditableDocContent, TDocsGenType, UUI3, UUI4,
 } from '../common';
 
 export class RatingDoc extends BaseDocsBlock {
     title = 'Rating';
+
+    // TODO: no such component in "@epam/uui"
+    override getDocsGenType = (): TDocsGenType => ('@epam/uui-components:RatingProps');
```

Comment:
get types from loveship package here. Because it's a skin specific component which will not be moved to the uui package.

Reply:
done